docker/base.dockerfile

```diff
@@ -51,12 +51,13 @@ RUN mkdir -p /usr/local/faasm/runtime_root/share
 WORKDIR /build/faasm
 
 # Build the basics here to set up the CMake build
+ARG FAASM_SGX_MODE
 RUN cmake \
     -GNinja \
     -DCMAKE_CXX_COMPILER=/usr/bin/clang++-13 \
     -DCMAKE_C_COMPILER=/usr/bin/clang-13 \
     -DCMAKE_BUILD_TYPE=Release \
-    -DFAASM_SGX_MODE=Simulation \
+    -DFAASM_SGX_MODE=${FAASM_SGX_MODE:-Simulation} \
```

Comment:
If `FAASM_SGX_MODE` is not set, use `Simulation`, [link to bash wiki](https://wiki.bash-hackers.org/syntax/pe#use_a_default_value).

Reply:
I always get confused with bash variable's expansion, so added the link and the explanation more for me than for you :rofl: